src/gt4py/next/program_processors/runners/dace/gtir_builtin_translators.py

```
@@ -76,13 +76,22 @@ class FieldopData:
         dc_node: DaCe access node to the data storage.
         gt_type: GT4Py type definition, which includes the field domain information.
         origin: List of start indices, in each dimension, for `FieldType` data.
-            Set to `None` only for `ScalarType` data.
+            Has to be `None` only for `ScalarType` data. For fields it is assumed
+            to be all zeros if not given.
```

This is not fully correct with the check below.
Assume you have a field type and pass `None` as `origin`.
Then `self.origin` will be set to `None` and the test in the `__post_init__()` function will fail.
But the doc string implies that it is silently converted to all zero.

Furthermore, something that I have not yet realized.
The class is frozen, so code such as:
```python
self.origin = 1
```
is invalid, but
```
self.origin.clear()
```
is valid.
Thus you should change the type to `Optional[tuple[dace.symbolic.SymbolicType, ...]]`.